activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -280,6 +280,26 @@ class TooManyRecords < ActiveRecordError
     #   member = Member.new
     #   member.avatar_attributes = {icon: 'sad'}
     #   member.avatar.width # => 200
+    #
+    # === Forms
+    #
+    # Use ActionView::Helpers::FormHelper#fields_for to create form elements
+    # for updating or destroying nested attributes.
+    #
+    # === Testing
+    #
+    # If you are using ActionView::Helpers::FormHelper#fields_for, your integration
+    # tests should replicate the HTML structure it provides. For example;
```

Comment:
Doesn't this always apply when using `accepts_nested_attributes_for`?
Even when you're not using `fields_for`?

Reply:
I'm also wondering if describing the integration with ActionView and ActionController is more suited for the guides.

Reply:
@p8 i agree with these suggestions but I won’t get to them for a little while. If you like, I’d love for you to take over the PR.

Reply:
@ghiculescu Sure! 😄